string variable # variable name
string value # value of variable